src/App.jsx

```diff
@@ -1,12 +1,18 @@
 import { Routes, Route } from 'react-router-dom';
 import './App.css';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
 import Library from './components/Library';
 import Categories from './components/Categories';
 import Header from './components/Header';
+import { recieveBooks } from './redux/books/books';
 
 function App() {
   const state = useSelector((state) => state);
+  const dispatch = useDispatch();
+  useEffect(() => async () => {
+    await dispatch(recieveBooks());
```

Comment:
- [ ] When you use `redux-thunk` you don't need `async` and `await` here, So please remove them and dispatch the action directly. See this [article ](https://redux.js.org/tutorials/fundamentals/part-6-async-logic#redux-async-data-flow)to know how thunk works